iFixit/react-commerce
# Fix related posts mobile view

This PR closes #83 

## Changelog

- Adds padding  for mobile version of related posts section


## QA

1. Visit the Vercel PR preview
2. Visit the parts collection page
3. Verify that mobile view of related news posts section has padding
